README.md

```
@@ -83,17 +86,50 @@ jobs:
 ### Custom Configuration
 
 ```yaml
 - uses: reactive-firewall/shellcheck-scan@v1
   with:
-    path: 'scripts/'
     severity: 'warning'
     shell-format: 'bash'
     match: '**/*.bash'
 ```
 
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)